modules/social_features/social_activity/modules/social_activity_filter/src/SocialActivityFilterOverride.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Drupal\social_activity_filter;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * Class SocialActivityFilterOverride.
+ *
+ * Configuration override.
+ *
+ * @package Drupal\social_event_type
+ */
+class SocialActivityFilterOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * Load overrides.
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+
+    // Override activity_stream views.
+    $config_name = 'views.view.activity_stream';
```

Comment:
@volodymyr-sydor-lemberg As discussed we need to support it for both: Home and Community streams.

Reply:
done